packages/d/digikam/pspec_x86_64.xml

```diff
@@ -48,7 +48,6 @@
             <Path fileType="library">/usr/lib64/qt6/plugins/digikam/bqm/Bqm_Convert16To8_Plugin.so</Path>
             <Path fileType="library">/usr/lib64/qt6/plugins/digikam/bqm/Bqm_Convert8To16_Plugin.so</Path>
             <Path fileType="library">/usr/lib64/qt6/plugins/digikam/bqm/Bqm_ConvertToAVIF_Plugin.so</Path>
-            <Path fileType="library">/usr/lib64/qt6/plugins/digikam/bqm/Bqm_ConvertToDNG_Plugin.so</Path>
```

Comment:
Why did this disappear?

Reply:
It looks like it become optional and needs `libjxl` on build time https://invent.kde.org/graphics/digikam/-/commit/e57659d386415b4dc3de2735e935bd0de7aafc72